### PR TITLE
Add capability to import Geant4 EM parameters

### PR DESCRIPTION
**Pull request by stognini**
This PR allows the import of EM parameters used by Geant4. These are global parameters stored in the `ImportData` struct and are the first step towards one of the goals outlined in issue #409 , which states that it is critical that we use the same model options that were used to generate Geant4 data.

Future PRs will update processes, such as the `BremsstrahlungProcess`, so that their options can be defined automatically instead of relying on a manual input.

### CODE REVIEW

File: test/celeritas/ext/RootImporter.test.cc [hunk 1]
```diff
@@ -371,3 +370,33 @@ TEST_F(RootImporterTest, volumes)
     EXPECT_VEC_EQ(expected_names, names);
     EXPECT_VEC_EQ(expected_solids, solids);
 }
+
+//---------------------------------------------------------------------------//
+TEST_F(RootImporterTest, em_params)
+{
+    const auto em_params = data_.em_params;
```
Comment (sethrj) [inline test/celeritas/ext/RootImporter.test.cc:377]:
```suggestion
    const auto& em_params = data_.em_params;
```
You accidentally made a copy :)

**Review by sethrj — COMMENTED** [review]
Changes look good with a few minor comments, plus it looks like `test/celeritas/data/simple-cms.root` grew by a factor of six. That seems way wrong for this small change; is it that we simply didn't update it after you added the elemental cross sections?

File: src/celeritas/ext/GeantImporter.cc [hunk 2]
```diff
@@ -381,10 +413,14 @@ GeantImporter::GeantImporter(const G4VPhysicalVolume* world) : world_(world)
 /*!
  * Construct by capturing a GeantSetup object.
  */
-GeantImporter::GeantImporter(GeantSetup&& setup) : setup_(std::move(setup))
+GeantImporter::GeantImporter(GeantSetup&& setup)
+    : setup_(std::move(setup))
+    , world_(G4TransportationManager::GetTransportationManager()
+                 ->GetNavigatorForTracking()
+                 ->GetWorldVolume())
```
[the remark this answers is not in the document]
Comment (stognini) [reply src/celeritas/ext/GeantImporter.cc:420]: Yes. The navigator ptr is set by `G4RunManagerKernel`, which sets up the world volume. If Geant4 was initialized correctly by the setup, at that point we should have a valid ptr. If not, we have the asserts.

**Review by sethrj — APPROVED** [review]
Nice @stognini ! Thanks!

**Pull request merged**